src/Evolution/DiscontinuousGalerkin/InboxTags.hpp

```diff
@@ -176,6 +179,26 @@ struct BoundaryCorrectionAndGhostCellsInbox {
       }
     }
   }
+
+  static std::string output_inbox(const type& inbox,
```

Comment:
Should we move these into `cpp` files whenever the template parameters of the tags are easy (`dim`)? Reduce compile time a bit

Reply:
Probably yes, but that requires some new files, cmake stuff, and instantiations. I'll try to do it later if I have time

Reply:
Sounds good